src/javascript/app/pages/user/accounts.js

```diff
@@ -244,9 +337,12 @@ const Accounts = (() => {
         }
     };
 
-    const showError = (localized_text) => {
-        $('#new_account_error').remove();
-        $('#new_account_opening').find('button').parent().append($('<p/>', { class: 'error-msg', id: 'new_account_error', text: localized_text }));
+    const showError = (localized_text, is_currency_change = false) => {
```

Comment:
Instead of passing a `is_currency_change ` boolean parameter maybe passing the ids would make this function more reusable and general?